client/rest-high-level/src/test/java/org/elasticsearch/client/SnapshotIT.java

```diff
@@ -49,13 +55,13 @@ private PutRepositoryResponse createTestRepository(String repository, String typ
             highLevelClient().snapshot()::createRepositoryAsync);
     }
 
-    private Response createTestSnapshot(String repository, String snapshot) throws IOException {
+    private Response createTestSnapshot(String repository, String snapshot, String index) throws IOException {
         Request createSnapshot = new Request("put", String.format(Locale.ROOT, "_snapshot/%s/%s", repository, snapshot));
         createSnapshot.addParameter("wait_for_completion", "true");
+        createSnapshot.setEntity(new NStringEntity("{\"indices\":\""+index+"\"}", ContentType.APPLICATION_JSON));
```

Comment:
you can also use setJsonEntity if you wish, that's handy

Reply:
Oh, neato